cluster/kubemark/gce/config-default.sh

```diff
@@ -96,9 +96,8 @@ CONTROLLER_MANAGER_TEST_ARGS="${KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS:-} ${CONTR
 SCHEDULER_TEST_ARGS="${KUBEMARK_SCHEDULER_TEST_ARGS:-} ${SCHEDULER_TEST_LOG_LEVEL} ${TEST_CLUSTER_API_CONTENT_TYPE} ${KUBEMARK_MASTER_COMPONENTS_QPS_LIMITS}"
 
 # Hollow-node components' test arguments.
-KUBELET_TEST_ARGS="--max-pods=100 $TEST_CLUSTER_LOG_LEVEL ${TEST_CLUSTER_API_CONTENT_TYPE}"
-KUBEPROXY_TEST_ARGS="${KUBEPROXY_TEST_LOG_LEVEL} ${TEST_CLUSTER_API_CONTENT_TYPE}"
-USE_REAL_PROXIER=${USE_REAL_PROXIER:-true}  # for hollow-proxy
+HOLLOW_KUBELET_TEST_ARGS="${HOLLOW_KUBELET_TEST_ARGS:-} ${KUBELET_TEST_LOG_LEVEL}"
```

Comment:
Can you also rename KUBELET_TEST_LOG_LEVEL -> HOLLOW_KUBELET_TEST_LOG_LEVEL (similarly for proxy)? Just so we're clear and consistent.